Hello Microverse Project

husush23 opened the pull request:
## Hello Microverse: first project

In this project, I:

- Added an index.html file that includes the header, "Hello Microverse!"
- Added a style.css file that includes a color style for the header
- Added a test.md file that should be ignored by git

ABDULALI3468 commented on .gitignore:
```diff
@@ -0,0 +1 @@
+node_modules/
```
It's a project requirement to add a `test.md file` in the `.gitignore file` as follows but currently it's not having any test.md file.

Consider making a file named `test.md`, add it in .gitignore file, and add the following text in it:- 

```
This file should be ignored by git
```


![image](https://user-images.githubusercontent.com/94223367/218786424-1a2b1780-4cc7-4776-add3-208bcd00701b.png)

ABDULALI3468 reviewed (changes requested):
# Changes Required 👍 

Hi @husush23 , 

<img src="https://media1.giphy.com/media/31Q36EHJ7C5ZoIMw79/giphy.gif"/>

Indeed you did an exceptional but still, there is room for changes to complete this project. 💯


## Highlights 
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.


 ## Required Changes ♻️

**1-** Please consider always using the name of your feature branch as a way to highlight what you did in it (e.g. 'created_sidebar' or 'added_comments_section')  In this case, your branch name is `first-project` which is not professional.

> Consider checking out from your current branch to a new branch and for this case, you can name it `feature-1/setup-basic-repo-configurations` or something else that describes what you did in the branch. Remember in the new PR which you will open for another review add the link of this PR in that PR's Description.


**2-** Please consider adding a professional title and description of your PR which describes what you have done in the Pull Request (PR). In this case, it's not describing it clearly.

>   ![pull_request_desc](https://user-images.githubusercontent.com/52934128/177452738-0610c008-a53a-454e-8aac-635cb4ed812c.gif)

**3-** In the case of README consider not leaving any of the sections in the [Microverse README](https://github.com/microverseinc/readme-template) template as it's a project requirement. Currently, you have all the sections but consider adding the content in it in accordance with your repository. Like tech stack needs to be one you used getting started guide needs to have details regarding your repo. Also, consider giving acknowledgments, etc, and don't forget to connect the MIT license file in README.

- The only optional section is FAQ.
- You can delete the live demo section if there is no deployment of the project on some server. In this case please consider deleting that section.
- Make sure the to add section Future Features, including 1-3 potential features to be added.


**4-** As it's not possible to update your commit messages but I request that next time please add a commit message in an imperative way and in the present tense.  

here are some examples of descriptive commit messages written in the imperative tense:

- Add a new feature to the dashboard
- Update error handling for the login form
- Refactor database connection code
- Remove deprecated method from API
- Fix the issue with the navigation menu not displaying properly

> Note that these commit messages clearly state what changes were made and are written in the present tense, which is the recommended style for Git commit messages. Using the imperative tense ("Add", "Update", "Refactor", etc.) helps to convey that the commit is making a specific change, rather than just describing a state.


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: ![GitHub followers](https://img.shields.io/github/followers/ABDULALI3468?label=ABDULALI&style=social)
> * This is My **Linkedin**: [![Linkedin Badge](https://img.shields.io/badge/-ABDUL%20ALI-blue?style=flat-square&logo=Linkedin&logoColor=white&link=https://www.linkedin.com/in/abdul-ali-5400bb216/)](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [![Twitter Badge](https://img.shields.io/badge/-@mrabdul_ali_-1ca0f1?style=flat-square&labelColor=1ca0f1&logo=twitter&logoColor=white&link=https://twitter.com/mrabdul_ali)](https://twitter.com/mrabdul_ali)&nbsp;

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

husush23 commented:
I accidentally deleted this pull request and I restored it later.

husush23 commented:
Hi, @ABDULALI3468 I have added a professional title and description of my PR and I have updated my readme.md file as requested. Also, I have updated my branch name from first-project to feature-1/setup-basic-repo-config on my local repo. However, I couldn't update the branch name on the remote repo because I attempted it earlier and it closed the pull request. Please I need your advice on if to go ahead and change the remote repo whilst closing the pull request or leave it this way.

Thanks

braincee commented on README.md:
````diff
@@ -55,234 +20,85 @@ After you're finished please remove all the comments and instructions!
- gem install rails
-```
- -->
+In order to run this project you need: only to clone this github repo
````
- [x] [OPTIOANL] Please add some prerequisites to make the README file look professional.
Example:

- Have some knowledge of HTML and CSS
- Be familiar with git commands
- Basics understanding of Linters

braincee commented on README.md:
```diff
@@ -55,234 +20,85 @@ After you're finished please remove all the comments and instructions!
-- [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
 <!-- PROJECT DESCRIPTION -->
```
- [x] [OPTIONAL] I kindly suggest you add a project description to make the README file look professional.

braincee commented on README.md:
````diff
@@ -55,234 +20,85 @@ After you're finished please remove all the comments and instructions!
-```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
-```
---->
+Clone this repository to your desired folder:git clone <the Repo url>
````
- [x] Please project setup commands for developers to clone the repository.
Example: 

```
cd my-folder
git clone https://github.com/husush23/Hello-Microverse.git
```

braincee commented on README.md:
````diff
@@ -55,234 +20,85 @@ After you're finished please remove all the comments and instructions!
-
-```sh
-  bin/rails test test/models/article_test.rb
-```
---->
+no installation needed, just clone is enough
````
- [ ] Please I suggest you add an installation command for other developers.
Example: `npm install / npm i`.

- [ ] Please I ensure you add run tests commands.
Example: `npx hint .`, `npx stylelint "**/*.{css,scss}"`

braincee reviewed (changes requested):
Hello @husush23  ✋, I am @braincee your Code Reviewer

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


![so-close](https://user-images.githubusercontent.com/78104544/225492039-ed22bba9-481a-489f-ae04-74623970af20.gif)

HIGHLIGHTS
------

 - GitHub flow rightfully used ✅
 - Meaningful title and summary for PR ✅
 - Professional README.md file ✅
 - No linter errors ✅

REQUIRED CHANGES ♻️
------
**_Please check the comments under the review._**

OPTIONAL CHANGES
--------
**_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @braincee in your question so I can receive the notification.

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

----

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Favourezeugwa reviewed (approved):
## STATUS: APPROVED 🥇 🏆 🍹 

Hi @husush23 ,

I am Favour 👩🏽 , your reviewer for this project milestone.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Yayyy! 💃🏽, Congratulations! 🎉 🎉🎉

![happy](https://user-images.githubusercontent.com/99246359/202491301-53f8c6a2-8ff4-4337-9b6b-9c75c8bdf345.gif)

## Highlights 💯 
- No Linter errors ✔️
- Good commit messages ✔️
- Professional Readme file ✔️
- Descriptive PR ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
